ur_ros2_control_demos/launch/test_forward_position_controller.launch.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2021 PickNik, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Denis Stogl
+
```

Comment:
I think these demo launch files should have a description of what they do at the top. Maybe a good description for this one would be:

`After a robot has been loaded, this will execute a linear motion.`

Reply:
If OK you can add those and merge PR if you are happy